src/archive.rs

```diff
@@ -7,11 +7,13 @@ use std::io::{self, SeekFrom};
 use std::marker;
 use std::path::Path;
 
+#[allow(unused_imports)]
```

Comment:
Is this attribute still necessary? If so I think that w/e is unused on wasi should probably get moved to w/e `#[cfg]`-specific code its used within.

Reply:
It was only used in one line of code so I removed the import instead and just fully qualified the function